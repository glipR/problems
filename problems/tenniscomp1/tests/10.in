256
98esnjwmc8z10889 532322537
5r7snw9mo3 503961866
dqrntcyqwudr41eudx 508292345
swu02ji9 187039259
y17loszvm1mx 322547301
01wxkokuw6yzi 975776600
b5oflwvxyqd 83962721
z7mpdglksk5c7 639863745
vmvi8w 420938364
l49ux45g 249161617
o3m3m508cwpv 410025240
1y7xh2a0dzrgk9e4h 6713738
ibxbisnv 82060476
zlpoi2dpm0yi42 504179966
zy26q8ljiy5z 901905629
9up6nl4s3727xuers 824764857
i938jc2b4nkpgrp15z 22697780
ri5saw64ode4pmy38j 975776600
a48wtc0z 599706967
oanw1v5c 801917611
2i7uerrtxdmcdc3 860834146
qixunqfl7psagbw0kvp 909631930
gb4h7 350023374
hrjfqtc35n 277134447
84mps90 170682749
6x6veulkm1y85f 838191697
k23ug4nz 23782399
qhgx9g691ci27 921401270
2bg1p4une4zwzx 570216886
7azg65u30ex0 301345160
3xako4aodfti 157262265
i1qwia3 948644886
sxmfaiwcwd1r 632605964
n6zf83ouekyxyv3i 608157630
6um70hh6 306529232
2vfete3nm5vp80x6a 814013875
7c1u88u 986635857
40i19 242755729
u6cyc3318r34v 830813637
slm30saok 120585020
vzd5nr56 238036763
8d2etnkwhz6 766957457
m6zi090lfmmglf 331566152
p9oiw8wheh9 212318347
ry3oeriga5bp6ujo 27884316
15p88a09wws 66077710
h3nq46sw1180ovm 415355094
sthwi0 480625101
56gm77q9 431248882
yz26e 390382071
8lc2g2ago 503617107
l8wkoyj5y91mm 441137445
jbm22czxaebvjk6nbtd 360872615
hcofgjbs722qbmka52 841575569
khsjvu 303593916
g7x8ctxp4vzzcrr2k 971476298
dmzza6astntp51zabo 148012651
dbmeh5q23yznvsb5obd 184729912
89b1cg41 117751356
d1qjrkkxhnbsp7o3 860369260
og68lddsd 599362084
m0hnbvvyfurgdm1 442892120
rpbmepqdhj 691879302
u3ztcraxdz8 324405191
kkxxmed 745539767
6dywmy89sc0s9ndhm4j 327218179
f997e 246866835
ijx51jk 607903858
ks5ahba 51891782
4rc0s83z 132983736
47i1sg9qlmasv49 950774648
not7bbgb 115937559
yhwjqmy 748642284
jly3xf0itxqjnf 644495153
dg6w54ca1c50l5p 573314397
bfqjms4imfz43zjrncw 405513321
1pilvfwkmp949 624888843
v0d8a1awhwiv34c 702528887
s5ilplerj 44360879
64pt4dmke0tudd6sn 599362084
vd5u7t8cmij9j 970880184
2okf7wo7a7blpcj7hyn 648388056
gyb7vntyc33n2s 610718734
8jhidleljgum75n6iw 651988756
l827favmiqj 652772100
unsqwezopkkz 648388056
07runou7ou5pb5qhm4q3 867376038
fj6wphz2dcu06sca 288586815
jfdpgswy 705462905
jze38 882958383
75gjqvrejx5oooafc9 270899494
kb3yj73eifp2nxpje 674498482
iqtgfalc2iylnu 188812950
rtjmvb0sby9 568203712
j5vo2j 828450696
ecsf9duyj3 362428182
jl0k7tp76xu 803499533
gmwnf7 364382101
rcu4an4ine5bu5uigr8 132983736
u752mj1s2s3cc261 194232530
v1wem 564374812
iaturp 283330028
rtnrt 653869063
wptzr0xzjb8s2815 156645154
fh3yave7yk827gq1xt 366478032
b5jnw5tc0 598863934
gawwzljlutsbxp1byd 56283673
f67a2jzgpqudzl1zg0bc 841895850
84p2su2 350023374
k6ubmf 111607761
m9p4zq002ig6cp 64784469
q6zowy 364382101
kkdxqqpj 836274430
rs9szxuxv94d7dbsme1 606662297
gsn2krw36igbleybnck 980406204
sanu0y5noidivd9z 172761324
4bh07uafmbxj 132983736
fuzvw 743065401
bqyb7a9ixccf2p3h 390611918
maym0bcj61ob7ckhgou 446545443
th0u1d2 900395783
ayw5jnvmmu5ddqrme7 159922782
uung73bolb9gpq4wkrt9 980436131
hyav4opd3q2 350023374
lhkxd 837491126
bo7dndswsta0 499615098
4hmmufkops56qaro2led 940449201
eze2gtvzyo 14399210
gcchtzwibs5 60851034
3d4fhth 51891782
dfywxp 386931309
opbbo7jt4ekue1ni6ohf 430862310
rkht6csman0vd6 696094585
al42qkxm 350754119
6e7yzv6vdkvl 693431963
8lzg8q7ls 568388442
jvjv4mmwmm1m4lgfx 319370359
syvm1r3nj32 797572766
28z06b 496699816
zevf99eza 36975860
ozcagbje 446989039
dwl6ch5l 846732944
g8wkn87eesikd 242755729
m4ygm69mxbcqa 359561040
4wmhuv 504179966
9vv2r 232269184
mhv808 836274430
fa09cpvcoz 929783042
aks8bmswcy2i8w1wcv 165335744
ggsk7qt2 860672346
9lqq6 437316767
6styjc6f1sowg 905376432
kg0t8nqhjon5brf9 351494893
srdok0twhms8t 36990933
khfyon4l9x4e6 51891782
6kdz0teawdfjwt2h 386931309
bm3eunskgod66e0l9h3p 386931309
2deatvljsvoj8f 415355094
z5ixl2g5o 179691439
be4egl4fycieji 868977232
a1988z 600263434
9rjkp63evnpa6zm530gq 137415792
h69vkdzr 306529232
cgtoa26 919838686
6lbur2 745102489
1xzvgcbo5g6md9c0l 613589333
mq8mtiuvhx7 978232864
mbi9xlhlds9 912326254
ji6gztcycgtby 146164383
3r7c16hsgdxl2 523645228
ldhoilm8xf 91262382
31vps9ul 599362084
bv7ul47f3m 257959407
czlbxvu 385061071
xa0ymgx203ylcg 725300935
jejirw2kjk86 905003100
ysxe64n5cf 62498088
xfqf7vmzm1lx5s9 426719331
vp09zg7hdtsio2p7 585072729
d686j01l1axvn4 553518403
lgnl6s 80143434
t6bbmj 581131722
q7xrss1 564374812
g82majil8e3umb6wgv 258282835
d18yg4neh 606662297
gkqgmky0vikyk5 364382101
e9dd1jtnoo5a2qxt 963249728
m4hxto 688577827
xdpwleu 624131522
1t5x0nyx0bi2v2rxhl6l 33785265
447szfzx07jav 789186270
49wdp1anlow9sl8or9qm 882958383
ee39d2pey8j6f0hp3lud 27884316
ce0d08wo93j7vzu8 909466030
x49o79pum 990959210
pk9iqy829 154862519
y62xplmghmld 774024417
6dx3gv9ul1jaeu5w 974312994
2numep 632560378
0sg94i2x3kyezidp22 43394172
14bai0xj6yr1b1jji 913659602
okpr5p2rqnfeevp5w4p 20276479
2tvakudaxb0pds 648207858
5g28wmvq7mchg365wra 288586815
rblmzz4azfy7y09j38 369573444
at4dbcrqnvxd3mlmmubc 825509201
kx8cjxuj1quxbo 447977984
smb8haetw8k2cz 700550786
zgmczh2khy5 83962721
xe3ichy0 441137445
lbh00sc3k9iv8ocxqa 568388442
l7tl4390k1kavoq6 35685815
myhxfdpwc20ze7ca7 12313114
4scte95zjglhe 351933332
qo6ac9fjna 713691327
1ahsdyu6h 254985653
eybqagp8vl78fo0qmln 421404694
1gtecqd83wjnktnl 480625101
pr8k9w0blas 281781099
4bkvkhnhhdcux 503617107
xda0lfm 850458478
h48fjf75 964212183
y5ui12 641128222
27xcs 146178790
z0hds7if6jwsu6k 13080915
muqxtcy 597103075
gzucnmmmepmca1eg 7260027
4wmgxav 836538238
i4tf48jb4ftims 735974635
ik9tkbwg6nfpr9mpsn 670901751
fzau4w6 693431963
q6cy2sfapk38gudakx 301345160
2re4l6mik4445 585072729
7839qjeitcm4r 705462905
q7ybfxmnys 465095996
8vx7hsr91eprhrmd 576824316
ayefbo1w2mksnz92ir 597103075
gp9njyjwpkcd 536545095
95lhtlcacfuvloz56yjl 68170483
98w2bqg 74950442
jui1rbaibpxrcdu8t 903331056
7z3yu03f 269753709
uy3wtyww3k1f 838191697
wqstzpx9frjyj0 628847517
c68wxq 656061664
uy4hjgf99mbrlf1 713691327
apskfnmmzs33uwul 113440585
u1t8blfrk6kzxnmyw 360872615
zr48pdxxqev 593958101
fwna6yzthe1czstw4a0 390611918
wlndkoq958 197615826
l9le2pw8xu 53055776
itmcnt 332545613
zzed5qf7r6hm9m3d 97595097
0m71k 495721728
ye0817b1h7v 612560376
