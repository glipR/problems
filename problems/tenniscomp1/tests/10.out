255
98esnjwmc8z10889 defeats 5r7snw9mo3
dqrntcyqwudr41eudx defeats swu02ji9
01wxkokuw6yzi defeats y17loszvm1mx
z7mpdglksk5c7 defeats b5oflwvxyqd
vmvi8w defeats l49ux45g
o3m3m508cwpv defeats 1y7xh2a0dzrgk9e4h
zlpoi2dpm0yi42 defeats ibxbisnv
zy26q8ljiy5z defeats 9up6nl4s3727xuers
ri5saw64ode4pmy38j defeats i938jc2b4nkpgrp15z
oanw1v5c defeats a48wtc0z
qixunqfl7psagbw0kvp defeats 2i7uerrtxdmcdc3
gb4h7 defeats hrjfqtc35n
6x6veulkm1y85f defeats 84mps90
qhgx9g691ci27 defeats k23ug4nz
2bg1p4une4zwzx defeats 7azg65u30ex0
i1qwia3 defeats 3xako4aodfti
sxmfaiwcwd1r defeats n6zf83ouekyxyv3i
2vfete3nm5vp80x6a defeats 6um70hh6
7c1u88u defeats 40i19
u6cyc3318r34v defeats slm30saok
8d2etnkwhz6 defeats vzd5nr56
m6zi090lfmmglf defeats p9oiw8wheh9
15p88a09wws defeats ry3oeriga5bp6ujo
sthwi0 defeats h3nq46sw1180ovm
56gm77q9 defeats yz26e
8lc2g2ago defeats l8wkoyj5y91mm
hcofgjbs722qbmka52 defeats jbm22czxaebvjk6nbtd
g7x8ctxp4vzzcrr2k defeats khsjvu
dbmeh5q23yznvsb5obd defeats dmzza6astntp51zabo
d1qjrkkxhnbsp7o3 defeats 89b1cg41
og68lddsd defeats m0hnbvvyfurgdm1
rpbmepqdhj defeats u3ztcraxdz8
kkxxmed defeats 6dywmy89sc0s9ndhm4j
ijx51jk defeats f997e
4rc0s83z defeats ks5ahba
47i1sg9qlmasv49 defeats not7bbgb
yhwjqmy defeats jly3xf0itxqjnf
dg6w54ca1c50l5p defeats bfqjms4imfz43zjrncw
v0d8a1awhwiv34c defeats 1pilvfwkmp949
64pt4dmke0tudd6sn defeats s5ilplerj
vd5u7t8cmij9j defeats 2okf7wo7a7blpcj7hyn
8jhidleljgum75n6iw defeats gyb7vntyc33n2s
l827favmiqj defeats unsqwezopkkz
07runou7ou5pb5qhm4q3 defeats fj6wphz2dcu06sca
jze38 defeats jfdpgswy
kb3yj73eifp2nxpje defeats 75gjqvrejx5oooafc9
rtjmvb0sby9 defeats iqtgfalc2iylnu
j5vo2j defeats ecsf9duyj3
jl0k7tp76xu defeats gmwnf7
u752mj1s2s3cc261 defeats rcu4an4ine5bu5uigr8
v1wem defeats iaturp
rtnrt defeats wptzr0xzjb8s2815
b5jnw5tc0 defeats fh3yave7yk827gq1xt
f67a2jzgpqudzl1zg0bc defeats gawwzljlutsbxp1byd
84p2su2 defeats k6ubmf
q6zowy defeats m9p4zq002ig6cp
kkdxqqpj defeats rs9szxuxv94d7dbsme1
gsn2krw36igbleybnck defeats sanu0y5noidivd9z
fuzvw defeats 4bh07uafmbxj
maym0bcj61ob7ckhgou defeats bqyb7a9ixccf2p3h
th0u1d2 defeats ayw5jnvmmu5ddqrme7
uung73bolb9gpq4wkrt9 defeats hyav4opd3q2
lhkxd defeats bo7dndswsta0
4hmmufkops56qaro2led defeats eze2gtvzyo
gcchtzwibs5 defeats 3d4fhth
opbbo7jt4ekue1ni6ohf defeats dfywxp
rkht6csman0vd6 defeats al42qkxm
6e7yzv6vdkvl defeats 8lzg8q7ls
syvm1r3nj32 defeats jvjv4mmwmm1m4lgfx
28z06b defeats zevf99eza
dwl6ch5l defeats ozcagbje
m4ygm69mxbcqa defeats g8wkn87eesikd
4wmhuv defeats 9vv2r
fa09cpvcoz defeats mhv808
ggsk7qt2 defeats aks8bmswcy2i8w1wcv
6styjc6f1sowg defeats 9lqq6
kg0t8nqhjon5brf9 defeats srdok0twhms8t
6kdz0teawdfjwt2h defeats khfyon4l9x4e6
2deatvljsvoj8f defeats bm3eunskgod66e0l9h3p
be4egl4fycieji defeats z5ixl2g5o
a1988z defeats 9rjkp63evnpa6zm530gq
cgtoa26 defeats h69vkdzr
6lbur2 defeats 1xzvgcbo5g6md9c0l
mq8mtiuvhx7 defeats mbi9xlhlds9
3r7c16hsgdxl2 defeats ji6gztcycgtby
31vps9ul defeats ldhoilm8xf
czlbxvu defeats bv7ul47f3m
jejirw2kjk86 defeats xa0ymgx203ylcg
xfqf7vmzm1lx5s9 defeats ysxe64n5cf
vp09zg7hdtsio2p7 defeats d686j01l1axvn4
t6bbmj defeats lgnl6s
q7xrss1 defeats g82majil8e3umb6wgv
d18yg4neh defeats gkqgmky0vikyk5
e9dd1jtnoo5a2qxt defeats m4hxto
xdpwleu defeats 1t5x0nyx0bi2v2rxhl6l
49wdp1anlow9sl8or9qm defeats 447szfzx07jav
ce0d08wo93j7vzu8 defeats ee39d2pey8j6f0hp3lud
x49o79pum defeats pk9iqy829
6dx3gv9ul1jaeu5w defeats y62xplmghmld
2numep defeats 0sg94i2x3kyezidp22
14bai0xj6yr1b1jji defeats okpr5p2rqnfeevp5w4p
2tvakudaxb0pds defeats 5g28wmvq7mchg365wra
at4dbcrqnvxd3mlmmubc defeats rblmzz4azfy7y09j38
smb8haetw8k2cz defeats kx8cjxuj1quxbo
xe3ichy0 defeats zgmczh2khy5
lbh00sc3k9iv8ocxqa defeats l7tl4390k1kavoq6
4scte95zjglhe defeats myhxfdpwc20ze7ca7
qo6ac9fjna defeats 1ahsdyu6h
1gtecqd83wjnktnl defeats eybqagp8vl78fo0qmln
4bkvkhnhhdcux defeats pr8k9w0blas
h48fjf75 defeats xda0lfm
y5ui12 defeats 27xcs
muqxtcy defeats z0hds7if6jwsu6k
4wmgxav defeats gzucnmmmepmca1eg
i4tf48jb4ftims defeats ik9tkbwg6nfpr9mpsn
fzau4w6 defeats q6cy2sfapk38gudakx
7839qjeitcm4r defeats 2re4l6mik4445
8vx7hsr91eprhrmd defeats q7ybfxmnys
ayefbo1w2mksnz92ir defeats gp9njyjwpkcd
98w2bqg defeats 95lhtlcacfuvloz56yjl
jui1rbaibpxrcdu8t defeats 7z3yu03f
uy3wtyww3k1f defeats wqstzpx9frjyj0
uy4hjgf99mbrlf1 defeats c68wxq
u1t8blfrk6kzxnmyw defeats apskfnmmzs33uwul
zr48pdxxqev defeats fwna6yzthe1czstw4a0
wlndkoq958 defeats l9le2pw8xu
itmcnt defeats zzed5qf7r6hm9m3d
ye0817b1h7v defeats 0m71k
98esnjwmc8z10889 defeats dqrntcyqwudr41eudx
01wxkokuw6yzi defeats z7mpdglksk5c7
vmvi8w defeats o3m3m508cwpv
zy26q8ljiy5z defeats zlpoi2dpm0yi42
ri5saw64ode4pmy38j defeats oanw1v5c
qixunqfl7psagbw0kvp defeats gb4h7
qhgx9g691ci27 defeats 6x6veulkm1y85f
i1qwia3 defeats 2bg1p4une4zwzx
2vfete3nm5vp80x6a defeats sxmfaiwcwd1r
7c1u88u defeats u6cyc3318r34v
8d2etnkwhz6 defeats m6zi090lfmmglf
sthwi0 defeats 15p88a09wws
8lc2g2ago defeats 56gm77q9
g7x8ctxp4vzzcrr2k defeats hcofgjbs722qbmka52
d1qjrkkxhnbsp7o3 defeats dbmeh5q23yznvsb5obd
rpbmepqdhj defeats og68lddsd
kkxxmed defeats ijx51jk
47i1sg9qlmasv49 defeats 4rc0s83z
yhwjqmy defeats dg6w54ca1c50l5p
v0d8a1awhwiv34c defeats 64pt4dmke0tudd6sn
vd5u7t8cmij9j defeats 8jhidleljgum75n6iw
07runou7ou5pb5qhm4q3 defeats l827favmiqj
jze38 defeats kb3yj73eifp2nxpje
j5vo2j defeats rtjmvb0sby9
jl0k7tp76xu defeats u752mj1s2s3cc261
rtnrt defeats v1wem
f67a2jzgpqudzl1zg0bc defeats b5jnw5tc0
q6zowy defeats 84p2su2
gsn2krw36igbleybnck defeats kkdxqqpj
fuzvw defeats maym0bcj61ob7ckhgou
uung73bolb9gpq4wkrt9 defeats th0u1d2
4hmmufkops56qaro2led defeats lhkxd
opbbo7jt4ekue1ni6ohf defeats gcchtzwibs5
rkht6csman0vd6 defeats 6e7yzv6vdkvl
syvm1r3nj32 defeats 28z06b
dwl6ch5l defeats m4ygm69mxbcqa
fa09cpvcoz defeats 4wmhuv
6styjc6f1sowg defeats ggsk7qt2
6kdz0teawdfjwt2h defeats kg0t8nqhjon5brf9
be4egl4fycieji defeats 2deatvljsvoj8f
cgtoa26 defeats a1988z
mq8mtiuvhx7 defeats 6lbur2
31vps9ul defeats 3r7c16hsgdxl2
jejirw2kjk86 defeats czlbxvu
vp09zg7hdtsio2p7 defeats xfqf7vmzm1lx5s9
t6bbmj defeats q7xrss1
e9dd1jtnoo5a2qxt defeats d18yg4neh
49wdp1anlow9sl8or9qm defeats xdpwleu
x49o79pum defeats ce0d08wo93j7vzu8
6dx3gv9ul1jaeu5w defeats 2numep
14bai0xj6yr1b1jji defeats 2tvakudaxb0pds
at4dbcrqnvxd3mlmmubc defeats smb8haetw8k2cz
lbh00sc3k9iv8ocxqa defeats xe3ichy0
qo6ac9fjna defeats 4scte95zjglhe
4bkvkhnhhdcux defeats 1gtecqd83wjnktnl
h48fjf75 defeats y5ui12
4wmgxav defeats muqxtcy
i4tf48jb4ftims defeats fzau4w6
7839qjeitcm4r defeats 8vx7hsr91eprhrmd
ayefbo1w2mksnz92ir defeats 98w2bqg
jui1rbaibpxrcdu8t defeats uy3wtyww3k1f
uy4hjgf99mbrlf1 defeats u1t8blfrk6kzxnmyw
zr48pdxxqev defeats wlndkoq958
ye0817b1h7v defeats itmcnt
01wxkokuw6yzi defeats 98esnjwmc8z10889
zy26q8ljiy5z defeats vmvi8w
ri5saw64ode4pmy38j defeats qixunqfl7psagbw0kvp
i1qwia3 defeats qhgx9g691ci27
7c1u88u defeats 2vfete3nm5vp80x6a
8d2etnkwhz6 defeats sthwi0
g7x8ctxp4vzzcrr2k defeats 8lc2g2ago
d1qjrkkxhnbsp7o3 defeats rpbmepqdhj
47i1sg9qlmasv49 defeats kkxxmed
yhwjqmy defeats v0d8a1awhwiv34c
vd5u7t8cmij9j defeats 07runou7ou5pb5qhm4q3
jze38 defeats j5vo2j
jl0k7tp76xu defeats rtnrt
f67a2jzgpqudzl1zg0bc defeats q6zowy
gsn2krw36igbleybnck defeats fuzvw
uung73bolb9gpq4wkrt9 defeats 4hmmufkops56qaro2led
rkht6csman0vd6 defeats opbbo7jt4ekue1ni6ohf
dwl6ch5l defeats syvm1r3nj32
fa09cpvcoz defeats 6styjc6f1sowg
be4egl4fycieji defeats 6kdz0teawdfjwt2h
mq8mtiuvhx7 defeats cgtoa26
jejirw2kjk86 defeats 31vps9ul
vp09zg7hdtsio2p7 defeats t6bbmj
e9dd1jtnoo5a2qxt defeats 49wdp1anlow9sl8or9qm
x49o79pum defeats 6dx3gv9ul1jaeu5w
14bai0xj6yr1b1jji defeats at4dbcrqnvxd3mlmmubc
qo6ac9fjna defeats lbh00sc3k9iv8ocxqa
h48fjf75 defeats 4bkvkhnhhdcux
4wmgxav defeats i4tf48jb4ftims
7839qjeitcm4r defeats ayefbo1w2mksnz92ir
jui1rbaibpxrcdu8t defeats uy4hjgf99mbrlf1
ye0817b1h7v defeats zr48pdxxqev
01wxkokuw6yzi defeats zy26q8ljiy5z
ri5saw64ode4pmy38j defeats i1qwia3
7c1u88u defeats 8d2etnkwhz6
g7x8ctxp4vzzcrr2k defeats d1qjrkkxhnbsp7o3
47i1sg9qlmasv49 defeats yhwjqmy
vd5u7t8cmij9j defeats jze38
f67a2jzgpqudzl1zg0bc defeats jl0k7tp76xu
uung73bolb9gpq4wkrt9 defeats gsn2krw36igbleybnck
dwl6ch5l defeats rkht6csman0vd6
fa09cpvcoz defeats be4egl4fycieji
mq8mtiuvhx7 defeats jejirw2kjk86
e9dd1jtnoo5a2qxt defeats vp09zg7hdtsio2p7
x49o79pum defeats 14bai0xj6yr1b1jji
h48fjf75 defeats qo6ac9fjna
4wmgxav defeats 7839qjeitcm4r
jui1rbaibpxrcdu8t defeats ye0817b1h7v
01wxkokuw6yzi defeats ri5saw64ode4pmy38j
7c1u88u defeats g7x8ctxp4vzzcrr2k
vd5u7t8cmij9j defeats 47i1sg9qlmasv49
uung73bolb9gpq4wkrt9 defeats f67a2jzgpqudzl1zg0bc
fa09cpvcoz defeats dwl6ch5l
mq8mtiuvhx7 defeats e9dd1jtnoo5a2qxt
x49o79pum defeats h48fjf75
jui1rbaibpxrcdu8t defeats 4wmgxav
7c1u88u defeats 01wxkokuw6yzi
uung73bolb9gpq4wkrt9 defeats vd5u7t8cmij9j
mq8mtiuvhx7 defeats fa09cpvcoz
x49o79pum defeats jui1rbaibpxrcdu8t
7c1u88u defeats uung73bolb9gpq4wkrt9
x49o79pum defeats mq8mtiuvhx7
x49o79pum defeats 7c1u88u
